usaspending_api/download/helpers.py

```diff
@@ -1,15 +1,13 @@
-import boto
+import boto3
 import csv
 import logging
 import math
-import mimetypes
-import multiprocessing
 import os
 import pandas as pd
+from boto3.s3.transfer import TransferConfig, S3Transfer
```

Comment:
Do you need this import line if you already import boto3?

Reply:
Fixing 👍